judo-ui-react/src/main/java/hu/blackbelt/judo/ui/generator/react/UiPageContainerHelper.java

```diff
@@ -109,6 +110,23 @@ public static String simpleActionDefinitionName(ActionDefinition actionDefinitio
         return relationName + (!relationName.isEmpty() ? firstToUpper(suffix) : suffix) + "Action";
     }
 
+    private static String getCallOperationName(ActionDefinition actionDefinition) {
+        OperationType operationType = null;
+        int index = 2;
+        if (actionDefinition instanceof CallOperationActionDefinition callOperationActionDefinition) {
+            operationType = callOperationActionDefinition.getOperation();
+        } else if (actionDefinition instanceof BulkCallOperationActionDefinition bulkCallOperationActionDefinition) {
+            operationType = bulkCallOperationActionDefinition.getOperation();
+        }
+        String actionDefinitionName = actionDefinition.getName();
+        String fallbackName = operationType.getName();
+        String[] nameParts = actionDefinitionName.split(NAME_SPLITTER);
+        if (nameParts.length >= index + 1) {
+            return nameParts[nameParts.length - index];
```

Comment:
What's the reason for the index variable?
```
int index = 2;
...
nameParts.length >= index + 1
...
nameParts.length - index
```

Why don't we simply use the following code?
```
 if (nameParts.length > 2) {
    return nameParts[nameParts.length - 2];
```

Reply:
Will be fixed later.